tests/test_wiki_images.py

```diff
@@ -2,36 +2,10 @@
 import json
 import pytest
 from app import app
+from unittest import mock
+from idunn.places import POI
 from starlette.testclient import TestClient
 
-INDICES = {
-    "admin": "munin_admin",
-    "street": "munin_street",
-    "addr": "munin_addr",
-    "poi": "munin_poi",
-}
-
-
-def load_place(file_name, mimir_client, doc_type):
-    index_name = INDICES.get(doc_type)
-
-    filepath = os.path.join(os.path.dirname(__file__), "fixtures", file_name)
-    with open(filepath, "r") as f:
-        place = json.load(f)
-        place_id = place["id"]
-        mimir_client.index(
-            index=index_name,
-            body=place,
-            doc_type=doc_type,  # 'admin', 'street', 'addr' or 'poi'
-            id=place_id,
-            refresh=True,
-        )
-
-
-@pytest.fixture(autouse=True)
-def load_all(mimir_client, init_indices):
-    load_place("orsay_museum.json", mimir_client, "poi")
-
```

Comment:
This code was unused as the fixtures are already defined globally in conftest.py